docs/Solana/DataFeeds/using-data-feeds-solana.md

```diff
@@ -45,6 +44,8 @@ Before you begin, set up your environment for development on Solana:
     node --version
```

Comment:
Same comment as above: Node version should be 14 or higher

Reply:
Done.